### PR TITLE
Moving typescripts dependencies at the root

### PR DESCRIPTION
**Pull request by julien51**
# Description

this should make our builds slightly faster and our images a bit smaller!

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/package.json [hunk 1]
```diff
@@ -40,12 +40,11 @@
     "@types/ethereumjs-util": "^5.2.0",
     "@types/express": "^4.16.1",
     "@types/express-serve-static-core": "^4.16.2",
+    "@types/jest": "^24.0.13",
     "@types/node": "^12.0.0",
     "@types/sequelize": "^4.27.44",
     "@types/stripe": "^6.25.10",
     "@types/supertest": "^2.0.7",
-    "@types/jest": "^24.0.13",
-    "@typescript-eslint/eslint-plugin": "^1.5.0",
```
Comment (akeem) [inline locksmith/package.json]: Was it intended to remove this from locksmith?

Comment (julien51) [reply locksmith/package.json]: yeah they are in the root.

Comment (julien51) [reply locksmith/package.json]: There is probably more cleanup to do as well. One step at a time ;)

**Comment by julien51** [comment]
I have to find the dependencies which are strictly used by the binaries `jest`, `typescript`... etc (which can be moved to the root and the ones which are loaded by the code directly.

**Pull request merged**